arXiv.org.js

```diff
@@ -650,9 +663,10 @@ var testCases = [
 				],
 				"date": "2018-10-10",
 				"abstractNote": "We introduce a new language representation model called BERT, which stands for Bidirectional Encoder Representations from Transformers. Unlike recent language representation models, BERT is designed to pre-train deep bidirectional representations from unlabeled text by jointly conditioning on both left and right context in all layers. As a result, the pre-trained BERT model can be fine-tuned with just one additional output layer to create state-of-the-art models for a wide range of tasks, such as question answering and language inference, without substantial task-specific architecture modifications. BERT is conceptually simple and empirically powerful. It obtains new state-of-the-art results on eleven natural language processing tasks, including pushing the GLUE score to 80.5% (7.7% point absolute improvement), MultiNLI accuracy to 86.7% (4.6% absolute improvement), SQuAD v1.1 question answering Test F1 to 93.2 (1.5 point absolute improvement) and SQuAD v2.0 Test F1 to 83.1 (5.1 point absolute improvement).",
-				"extra": "arXiv: 1810.04805\nversion: 1",
+				"extra": "arXiv:1810.04805 [cs]\nversion: 1\ntype: article",
+				"institution": "arXiv",
 				"libraryCatalog": "arXiv.org",
-				"publicationTitle": "arXiv:1810.04805 [cs]",
+				"reportNumber": "arXiv: 1810.04805",
```

Comment:
```suggestion
				"reportNumber": "arXiv:1810.04805",
```

There shouldn't be a space here should there?